README.md

```diff
@@ -103,7 +101,7 @@ Before a new major release master is branched for LTS releases of the prior majo
 
 Additionally there are branches for stable release lines prior to 1.0 of minor versions. Example: `0.8.x`, `0.10.x`, `0.12.x`.
 
-All release branches other than master are managed by the LTS Working Group and fall under their LTS release policy.
+All branches other than master are managed by the LTS Working Group and fall under the LTS release policy.
```

Comment:
We could have feature branches or branches that are trying unstable versions of v8, and those wouldn't be handled by LTS, that's why this was originally "release" branches.